genic/main.c

```diff
@@ -58,13 +58,25 @@ int main(int argc, char **argv)
   if(0 != big_file_mpi_create(&bf, buf, MPI_COMM_WORLD)) {
       endrun(0, "%s\n", big_file_get_error_message());
   }
-  saveheader(&bf, TotNumPart);
+  /*Massive neutrinos*/
+
+  const int64_t TotNu = (int64_t) NGridNu*NGridNu*NGridNu;
+  double total_nufrac = 0;
+  if(TotNu > 0) {
+    const double kBMNu = 3*CP.ONu.kBtnu / (CP.MNu[0]+CP.MNu[1]+CP.MNu[2]);
+    double v_th = NU_V0(InitTime, kBMNu, UnitVelocity_in_cm_per_s);
+    if(!UsePeculiarVelocity)
+        v_th /= sqrt(InitTime);
+    total_nufrac = init_thermalvel(v_th, Max_nuvel/v_th, 0);
```

Comment:
Is this the total fraction of massive neutrino? Would it be useful to save this value in the header as a properly named attr?

Reply:
Yes! This is the fraction of massive neutrino in particles.